.github/workflows/continuous-integration.yml

```diff
@@ -84,7 +84,15 @@ jobs:
         id: matrix_generation
         env:
           PROJECT_NAME_TO_TEST: ${{ matrix.projectName }}
-        run: cd tests/${PROJECT_NAME_TO_TEST} && docker run -i --entrypoint "/action/main.js" -v $(realpath .):/github/workspace -w=/github/workspace ${TEST_TAG} $(test -r diff && cat diff || echo -n "")
+        run: |
+          cd tests/${PROJECT_NAME_TO_TEST} && \
+          docker run \
+            -i \
+            --entrypoint "/action/main.js" \
+            -v $(realpath .):/github/workspace \
+            --env-file=test.env \
```

Comment:
I added a `test.env` to each test directory: ugly, but IMO better than adding more conditional logic here